StructuralPreprocessing/StructuralPreprocessing.XNAT_MARK_RUNNING_STATUS

```diff
@@ -49,10 +49,14 @@ get_options()
 	local arguments=($@)
 
 	# initialize global output variables
-	unset g_project
-	unset g_subject
-	unset g_classifier
-	unset g_running
+    unset g_user
+    unset g_password
+    unset g_server
+    unset g_project
+    unset g_subject
+    unset g_session
```

Comment:
Same comment as for FunctionalPreprocessing version of this file. Let's continue to pass in the `classifier=<value>` argument and build the `g_session` value based on the `g_subject` value and the `g_classifier` value concatenated together.

Reply:
changed to g_classifier